admin-dev/themes/new-theme/public/theme.rtlfix

```diff
@@ -132,3 +132,8 @@
   border-right-style: none;
   border-left-style: solid;
 }
+
+/* Fix scrollbar position in sidebar menu */
```

Comment:
```suggestion
```

Reply:
I would let it, not that bad because it's not that understandable